pyrasterframes/python/examples/__init__.py

```diff
@@ -1,15 +1,24 @@
 # examples_setup
-from pathlib import Path
+import os
+import glob
 from pyspark.sql import SparkSession
 
 __all__ = ["resource_dir", "example_session"]
 
-jarpath = list(Path('../target').resolve().glob('**/pyrasterframes*.jar'))
-if len(jarpath) > 0:
-    pyJar = jarpath[0].as_uri()
+HERE = os.path.dirname(os.path.realpath(__file__))
 
-    def example_session():
-        return (SparkSession.builder
+scala_target = os.path.realpath(os.path.join(HERE, '..', '..', 'scala-2.11'))
+resource_dir = os.path.realpath(os.path.join(scala_target, 'test-classes'))
+jarpath = glob.glob(os.path.join(scala_target, 'pyrasterframes-assembly*.jar'))
+
+if not len(jarpath) == 1:
+    raise RuntimeError("Expected to find exactly one assembly. Found: ", jarpath)
```

Comment:
Give advice on what to do if that's true. `sbt clean` ?

Reply:
And or need to build the jars? See pyspark helpful message https://github.com/apache/spark/blob/master/python/setup.py#L43-L52